### PR TITLE
Implement new payment: Cashalo

### PR DESCRIPTION
**Pull request by andykim**
## Description

Implement new payment: Cashalo

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Cosmetic change (text changing or color changing)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code (if necessary), particularly in hard-to-understand areas
- [ ] My changes generate no new vulnerabilities
- [ ] I have added tests that prove my fix is effective or that my feature works

## Screenshot

Screenshot from 2022-03-25 13-31-26
Screenshot from 2022-03-25 13-30-46

### CODE REVIEW

File: Xendit/M2Invoice/composer.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "xendit/m2invoice",
   "description": "Xendit Payment Gateway Module",
   "type": "magento2-module",
-  "version": "3.5.0",
+  "version": "3.5.1",
```
Comment (IreneGohtami) [inline Xendit/M2Invoice/composer.json:5]: please change the minor version, coz we're adding new payment channel: 3.6.0

File: Xendit/M2Invoice/etc/config.xml [hunk 2]
```diff
@@ -301,6 +301,20 @@
...
                 <currency>PHP</currency>
             </billease>
+            <cashalo>
+                <active>1</active>
+                <payment_action>initialize</payment_action>
+                <model>Xendit\M2Invoice\Model\Payment\Cashalo</model>
+                <title>Cashalo</title>
+                <description>Pay for orders using Cashalo via Xendit</description>
+                <min_order_total>1500</min_order_total>
+                <max_order_total>4500</max_order_total>
```
Comment (IreneGohtami) [inline Xendit/M2Invoice/etc/config.xml:311]: where did you get this min & max info from? I can't find it in https://xendit.atlassian.net/wiki/spaces/INV/pages/1458733177/Invoice+Payment+Channel+List

Comment (andykim) [reply Xendit/M2Invoice/etc/config.xml:311]: https://xendit.atlassian.net/wiki/spaces/INV/pages/2546795082/Payment+methods => @IreneGohtami You can find it here

File: Xendit/M2Invoice/composer.json (hunk 1 above)
Comment (andykim) [reply Xendit/M2Invoice/composer.json]: @IreneGohtami I changed it. plz check

File: Xendit/M2Invoice/etc/config.xml (hunk 2 above)
Comment (IreneGohtami) [reply Xendit/M2Invoice/etc/config.xml:311]: ok thanks! 4500 is quite low for a maximum, but if that's the case then ok. @andykim

**Review by IreneGohtami — APPROVED** [review]

**Review by yanoandri — APPROVED** [review]

**Pull request merged**